spec/unit/dry/inflector/pluralize_spec.rb

```diff
@@ -8,6 +8,12 @@
       end
     end
 
+    Fixtures::Pluralize.irregular.each do |singular, plural|
+      it "#{singular} => #{plural} is idempotent" do
+        expect(subject.pluralize(i(singular))).to eq(plural)
```

Comment:
@abinoam I don't understand the purpose of this line. Isn't it identical to the existing tests for regular pluralizations?